rust-iml.spec

```diff
@@ -93,6 +93,7 @@ Obsoletes: iml-device-scanner-proxy
 
 %post agent
 systemctl preset rust-iml-agent.path
+%tmpfiles_create %{_tmpfilesdir}/iml-agent.conf
```

Comment:
I haven't looked at what current in CentOS 7.7, but looks like this is deprecated upstream at least:

https://github.com/systemd/systemd/blob/master/src/core/macros.systemd.in#L92-L96

Reply:
for our version, `%tmpfiles_create` is the only option available.